home/views.py

```diff
@@ -87,9 +87,10 @@ def new_question(request):
                 tags_error_msg = '''Invalid data.
              You may enter up to 5 tags which are separated by ",".
              Each tag must have between 2-20 characters.'''
-                messages.error(request, tags_error_msg)
+                messages.error(request, {'title': 'ERROR:', 'message_content': tags_error_msg})
```

Comment:
Ok, great addition!
I think you should add a test for the title as well.

Reply:
ok will do